client/my-sites/google-my-business/stats/chart.js

```diff
@@ -0,0 +1,113 @@
+/** @format */
+
+/**
+ * External dependencies
+ */
+import PropTypes from 'prop-types';
+import React, { Component } from 'react';
+import { connect } from 'react-redux';
+import { isEqual } from 'lodash';
+
+/**
+ * Internal dependencies
+ */
+import Card from 'components/card';
+import CardHeading from 'components/card-heading';
+import FakeData from './fake-data';
+import PieChart from 'components/pie-chart';
+import PieChartLegend from 'components/pie-chart/legend';
+import SectionHeader from 'components/section-header';
+import { changeGoogleMyBusinessStatsInterval } from 'state/google-my-business/actions';
+import { getInterval } from 'state/google-my-business/selector';
+import { getSelectedSiteId } from 'state/ui/selectors';
+
+class GoogleMyBusinessStatsChart extends Component {
+	static props = {
+		changeGoogleMyBusinessStatsInterval: PropTypes.func.isRequired,
+		chartTitle: PropTypes.oneOfType( [ PropTypes.func, PropTypes.string ] ),
+		data: PropTypes.array.isRequired,
+		dataSeriesInfo: PropTypes.object,
+		description: PropTypes.string,
+		interval: PropTypes.string.isRequired,
+		siteId: PropTypes.number.isRequired,
+		statType: PropTypes.string.isRequired,
+		title: PropTypes.string.isRequired,
+	};
+
+	static defaultProps = {
+		dataSeriesInfo: {},
+	};
+
+	shouldComponentUpdate( nextProps ) {
+		return (
+			this.props.interval !== nextProps.interval || ! isEqual( this.props.data, nextProps.data )
+		);
+	}
+
+	transformData( data ) {
+		return data.map( value => {
+			return {
+				value: value.dimensionalValues.value,
+				description:
+					this.props.dataSeriesInfo[ value.metric ] !== undefined
```

Comment:
What you think about lodash's `get( this.props.dataSeriesInfo, value.metric + '.description', '' )` instead of the ternary here and below?